searx/templates/simple/answer/weather.html

```diff
@@ -0,0 +1,63 @@
+{% macro show_weather_data(answer, data) %}
+  <table>
+    <colgroup>
+      <col span="1" class="thumbnail">
+      <col span="1" class="title">
+      <col span="1" class="measured">
+      <col span="1" class="title">
+      <col span="1" class="measured">
+      <col>
+    </colgroup>
+    <tbody>
+      <tr>
+        <td rowspan="4">
+          {%- if data.url %}<img class="symbol" src="{{ data.url }}" title="{{ data.summary }}">{% endif -%}
+        </td>
+      </tr>
+      <tr>
+        <td>{{ _("Temperature") }}:</td>
+        <td>{{ data.temperature.l10n(locale=data.location) }}</td>
+        <td>{{ _("Feels Like") }}:</td>
+        <td>{{ data.feels_like.l10n(locale=data.location) }}</td>
+      </tr>
+      <tr>
+        <td>{{ _("Wind") }}:</td>
+        <td>{{ data.wind_from.l10n(locale=data.location) }}: {{ data.wind_speed.l10n(locale=data.location) }}</td>
+        <td>{{ _("Pressure") }}:</td>
+        <td>{{ data.pressure.l10n(locale=data.location) }}</td>
+      </tr>
+      <tr>
+        <td>{{_("Humidity")}}:</td>
+        <td>{{ data.humidity.l10n(locale=data.location) }}</td>
+        <td></td>
+        <td></td>
+      </tr>
```

Comment:
Do we really want to make all these mandatory? I don't think all weather engines have pressure data (e.g. wttr.in I think).